README.md

```diff
@@ -1,2 +1,161 @@
-# slack-cli
-Practice working with APIs by writing a command line application that interacts with the Slack API
+# Project Title
```

Comment:
Are they supposed to put their project title here?  Or is this a placeholder for instructors?